ui/components/app/srp-quiz-popover/QuizContent/QuizContent.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { BUTTON_SIZES, Button, Text } from '../../../component-library';
+import Box from '../../../ui/box';
+import { IQuizInformationProps } from '../types';
+
+export default function QuizContent({
+  icon,
+  image,
+  content,
+  moreContent,
+  buttons,
+}: IQuizInformationProps) {
+  return (
+    <>
+      {icon && (
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Row}
+          alignItems={AlignItems.center}
+          justifyContent={JustifyContent.center}
+        >
+          {icon}
+        </Box>
+      )}
+      {image && <img src={image} width="100%" />}
+      <Text
+        variant={TextVariant.bodyLgMedium}
+        textAlign={TextAlign.Center}
+        margin={2}
+        color={icon?.props.color} // Inherit this text color from the icon's color
```

Comment:
`TextColor` has an `inherit` value

Reply:
Whoops, "inherit" was a confusing word to write in the comment.  It's not inheriting the color of its parent, but the color of its sibling.  So maybe I should change the comment to say "copy."